src/css/application.css

```diff
@@ -604,3 +604,21 @@ body {
 .u__hidden {
   display: none;
 }
+
+.loading {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+}
+
+.loading__spinner {
+  background-image: url('/images/spinner.gif');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
Assuming there isn’t a disadvantage I’m not thinking of, my preference is to use shorthand properties like `background`